WcaOnRails/app/models/wcif.rb

```diff
@@ -13,6 +13,23 @@ def json
     {
       "formatVersion" => "1.0",
       "id" => competition.id,
+      "events" => competition.competition_events.map(&method(:competition_event_to_json)),
```

Comment:
That would work if the CompetitionEvent class had a method called `competition_event_to_json`, but it does not. We *could* add such a method though, it just would not live in this file.